persiancalendar/src/main/res/values-fa-rIR/strings.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Persian Calendar</string>
+    <string name="pcv_shanbe">ش</string>
```

Comment:
I would go with more language agnostic names such as: `pcv_persian_day_1`, `pcv_persian_month_10`...

Reply:
oh... I just saw the other strings.xml file 
```
 <string name="pcv_shanbe">Sat</string>
    <string name="pcv_yekshanbe">Sun</string>
    <string name="pcv_doshanbe">Mon</string>
    <string name="pcv_seshanbe">Tue</string>
    <string name="pcv_chaharshanbe">Wed</string>
    <string name="pcv_panjshanbe">Thu</string>
    <string name="pcv_jome">Fri</string>
```

don't use "finglish" naming, please.

Reply:
this file is: persiancalendar/src/main/res/values-fa-rIR/strings.xml

please consider changing the parent folder to `values-fa` instead of `values-fa-rIR` so it will still work in farsi in other farsi speaking regions.